apps/condo/domains/common/utils/apolloErrorFormatter.js

```diff
@@ -29,17 +29,55 @@ const { ApolloError, AuthenticationError } = require('apollo-server-errors')
 const { GraphQLError, printError } = require('graphql')
 
 const ensureError = require('ensure-error')
-const { serializeError } = require('serialize-error')
-const cuid = require('cuid')
-const { pick, pickBy, identity, toArray, _, toString, get } = require('lodash')
-
-const { graphqlLogger } = require('@keystonejs/keystone/lib/Keystone/logger')
+const { pick, pickBy, identity, toArray, _, toString, get, set, isArray } = require('lodash')
 
 const conf = require('@core/config')
 
 const IS_HIDE_INTERNALS = conf.NODE_ENV === 'production'
+const COMMON_ERROR_CASES = {}
+
+function _getAllErrorMessages (error) {
```

Comment:
Why _ notation?

Reply:
it's a privat. Someone can add it exports. I don't want it.